packages/graphql-docs-generator/awsAppSyncDirectives.graphql

```diff
@@ -19,4 +19,5 @@ directive @aws_auth(cognito_groups: [String!]!) on FIELD_DEFINITION
 directive @aws_api_key on FIELD_DEFINITION | OBJECT
 directive @aws_iam on FIELD_DEFINITION | OBJECT
 directive @aws_oidc on FIELD_DEFINITION | OBJECT
+directive @aws_lambda on FIELD_DEFINITION | OBJECT
```

Comment:
Oh can you move this up a line (before aws_oidc) for consistency.

Reply:
Updated.